mico-core/src/main/java/io/github/ust/mico/core/service/MicoStatusService.java

```diff
@@ -216,30 +216,32 @@ public MicoServiceStatusResponseDTO getServiceStatus(MicoService micoService) {
     }
 
     public List<MicoServiceInterfaceStatusResponseDTO> getServiceInterfaceStatus(@NotNull MicoService micoService,
-                                                                                 @NotNull List<String> errorMessages) {
+                                                                                 @NotNull List<MicoMessageResponseDTO> errorMessages) {
         List<MicoServiceInterfaceStatusResponseDTO> interfacesInformation = new ArrayList<>();
         for (MicoServiceInterface serviceInterface : micoService.getServiceInterfaces()) {
             String interfaceName = serviceInterface.getServiceInterfaceName();
             List<String> publicIps = new ArrayList<>();
+            String msg;
             try {
                 Optional<Service> kubernetesServices = micoKubernetesClient.getInterfaceByNameOfMicoService(micoService, interfaceName);
                 if (kubernetesServices.isPresent()) {
                     publicIps = getPublicIpsOfKubernetesService(kubernetesServices.get());
                     if (publicIps.isEmpty()) {
-                        errorMessages.add("There is no Kubernetes service for the interface '" +
-                            interfaceName + "' of MicoService '" + micoService.getShortName() + "' '" + micoService.getVersion() + "'.");
+                        msg = "There are no public IP addresses available yet for the interface '" + interfaceName
+                            + "' of MicoService '" + micoService.getShortName() + "' '" + micoService.getVersion() + "'.";
+                        log.debug(msg);
+                        errorMessages.add(new MicoMessageResponseDTO(msg, Type.ERROR));
```

Comment:
Consider to use `Type.INFO` or `Type.WARNING` instead of `Type.ERROR`. After a deployment it needs some seconds until the public IP addresses for a Kubernetes service are available. That should be not considered as an error!